src/Microsoft.FeatureManagement/Targeting/AudienceExclusion.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using System.Collections.Generic;
+
+namespace Microsoft.FeatureManagement.FeatureFilters
+{
+	/// <summary>
+	/// Excludes users and groups from the audience.
+	/// </summary>
+	public class AudienceExclusion
```

Comment:
The name is a bit problematic for me. It's really just a simplified audience definition. Nothing about the properties make it `exclusion` related.

Reply:
Completely agree. I was deciding if the library leaned towards generic or more specific object oriented. I saw TargetingContext and took that as leaning more towards specificity (but that's not a straightforward comparison). That, and, I couldn't land on a good name for a generic version.

What do you prefer?
`AudienceCompact`
`SimplifiedAudience`
`BasicAudience`
`StandardAudience`

Reply:
I also thought of `SimplifiedAudience` `BasicAudience`. I also considered what is different between this and our existing `Audience`. It's the absence of percentages. I then considered `DeterministicAudience`.

Reply:
I thought the same thing with absence of percentages but landed on `AudienceWithoutRollout`. `DeterministicAudience` is cleaner. 

I'm leaning towards `BasicAudience` because I think it's the most intuitive and least future-restrictive. I'll go ahead and make changes towards that. Let me know if you feel strongly about one of the other options.